libflux/flux-core/src/formatter/tests.rs

```diff
@@ -1268,3 +1268,77 @@ a",
     );
     assert_unchanged(r#"a = "a string literal with a tabstop '	'""#);
 }
+
+
+#[test]
+fn consistent_multiline_formatting() {
+    assert_unchanged(
+        r#"import "array"
+
+array.from(
+    rows: [
+        {_value: "one", _time: 2021-08-30T00:00:00Z},
+        {_value: "one", _time: 2021-08-30T00:00:00Z},
+        {_value: "one", _time: 2021-08-30T00:00:00Z},
+        {_value: "one", _time: 2021-08-30T00:00:00Z},
+        {_value: "one", _time: 2021-08-30T00:00:00Z},
+        {_value: "one", _time: 2021-08-30T00:00:00Z},
+        {_value: "one", _time: 2021-08-30T00:00:00Z},
+    ],
+    foo: false,
+)"#,
+    );
+
+    assert_format(
+        r#"import "array"
+
+array.from(rows: [{_value: "one", _time: 2021-08-30T00:00:00Z}, {_value: "one", _time: 2021-08-30T00:00:00Z}, {_value: "one", _time: 2021-08-30T00:00:00Z},
+    {_value: "one", _time: 2021-08-30T00:00:00Z},
+    {_value: "one", _time: 2021-08-30T00:00:00Z},
+    {_value: "one", _time: 2021-08-30T00:00:00Z},
+    {_value: "one", _time: 2021-08-30T00:00:00Z},
+], foo: false)"#,
+    r#"import "array"
+
+array.from(
+    rows: [
+        {_value: "one", _time: 2021-08-30T00:00:00Z},
+        {_value: "one", _time: 2021-08-30T00:00:00Z},
+        {_value: "one", _time: 2021-08-30T00:00:00Z},
+        {_value: "one", _time: 2021-08-30T00:00:00Z},
+        {_value: "one", _time: 2021-08-30T00:00:00Z},
+        {_value: "one", _time: 2021-08-30T00:00:00Z},
+        {_value: "one", _time: 2021-08-30T00:00:00Z},
+    ],
+    foo: false,
+)"#
+    );
+
+    assert_format(
```

Comment:
Something that is formatted should never change (ie format is idempotent) so we could add an `assert_unchanged` check on the formatted ouput in `fn assert_format`.